### PR TITLE
Made some clear work with design, added bar at top of mail content

### PR DESCRIPTION
**Pull request by Akumai**

### CODE REVIEW

File: src/components/MailBar.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+            <li>
+                <SvgButton path="M 23.707031 10.792969 L 14.710938 1.800781 C 14.425781 1.515625 13.996094 1.425781 13.621094 1.582031 C 13.25 1.734375 13.007812 2.101562 13.007812 2.503906 L 13.007812 6.507812 C 7.253906 6.742188 2.238281 10.625 0.632812 16.191406 C -0.0625 18.601562 -0.0117188 20.5625 0.0117188 21.300781 L 0.015625 21.496094 C 0.015625 21.945312 0.316406 22.339844 0.746094 22.457031 C 0.835938 22.480469 0.925781 22.492188 1.011719 22.492188 C 1.359375 22.492188 1.6875 22.3125 1.871094 22.007812 C 5.832031 15.351562 11.074219 15.164062 13.007812 15.351562 L 13.007812 20.496094 C 13.007812 20.898438 13.25 21.265625 13.621094 21.417969 C 13.996094 21.574219 14.425781 21.488281 14.710938 21.203125 L 23.707031 12.207031 C 24.097656 11.816406 24.097656 11.183594 23.707031 10.792969 Z M 23.707031 10.792969"
+                    svgClasses={BTN_SVG_CLASSES}
+                    aClasses={BTN_CLASSES} fill="true" />
+            </li>
+        </ul>
+    )
+};
+
+export default MailBar;
```
Comment (kudanov) [inline src/components/MailBar.js]: add newLine pls

File: src/App.css [hunk 2]
```diff
@@ -1,3 +1,24 @@
...
+
+/* Handle */
+::-webkit-scrollbar-thumb {
+    background: #a1a1a1;
+}
+
+/* Handle on hover */
+::-webkit-scrollbar-thumb:hover {
+    background: #737373;
+}
```
Comment (kudanov) [inline src/App.css]: new line

File: src/components/MailList.js [hunk 3]
```diff
@@ -1,23 +1,25 @@
...
       </div>
     </div>
   );
 }
 
 const MailList = ({store}) => (
-  <div class="bg-yellow-200 w-80 overflow-auto">
-    <div class="bg-yellow-300 h-12 sticky top-0">
-      <SearchPanel />
+  <div class="w-2/5 px-6 overflow-y-auto border-gray-200">
```
Comment (kudanov) [inline src/components/MailList.js:20]: overflow-x-hidden?

File: src/components/NavPanel/index.js [hunk 4]
```diff
@@ -1,40 +1,27 @@
...
-        </a>
-    );
-};
-
-const NavPanel = () => (
+const NavPanel = ({store}) => (
     <>
-        <style>{style}</style>
-        <div class="bg-gray-300 w-16 flex flex-col min-h-full relative">
+        <div class="bg-gray-800 flex flex-col min-h-full relative">
```
Comment (kudanov) [inline src/components/NavPanel/index.js:7]: can we use this classes in ul and remove unnecessary div?

File: package.json [hunk 5]
```diff
@@ -37,7 +41,11 @@
   "main": "public/electron.js",
   "homepage": "./",
   "build": {
-    "appId": "com.electron.${name}",
```
Comment (kudanov) [inline package.json:40]: it must work.

File: src/components/NavPanel/index.js [hunk 6]
```diff
@@ -1,40 +1,27 @@
...
             <ul>
                 <li>
-                    <SvgButton path="M3 19v-8.93a2 2 0 01.89-1.664l7-4.666a2 2 0 012.22 0l7 4.666A2 2 0 0121 10.07V19M3 19a2 2 0 002 2h14a2 2 0 002-2M3 19l6.75-4.5M21 19l-6.75-4.5M3 10l6.75 4.5M21 10l-6.75 4.5m0 0l-1.14.76a2 2 0 01-2.22 0l-1.14-.76" />
+                    <SvgButton path="M3 19v-8.93a2 2 0 01.89-1.664l7-4.666a2 2 0 012.22 0l7 4.666A2 2 0 0121 10.07V19M3 19a2 2 0 002 2h14a2 2 0 002-2M3 19l6.75-4.5M21 19l-6.75-4.5M3 10l6.75 4.5M21 10l-6.75 4.5m0 0l-1.14.76a2 2 0 01-2.22 0l-1.14-.76" 
+                        onClick={() => store.setInboxActive()}
+                        aClasses={ BTN_CLASSES }/>
```
Comment (kudanov) [inline src/components/NavPanel/index.js]: replace with divClasses

File: src/components/SvgButton/index.js [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+            <svg class={(svgClasses ?? DEFAULT_SVG_CLASSES) + " " + "stroke-1"} xmlns="http://www.w3.org/2000/svg" fill={fill ? "currentColor" : "none"} viewBox="0 0 24 24" stroke={fill ? "none" : "currentColor"}>
+                {Array.isArray(path)
+                    ? path.map(d => formatPath(d))
+                    : formatPath(path)}
+            </svg>
+        </div>
+    );
+};
+
+export default SvgButton;
```
Comment (kudanov) [inline src/components/SvgButton/index.js]: new line

File: src/utils/Store.js [hunk 8]
```diff
@@ -40,6 +19,8 @@ export const createStore = () => {
         state,
         reset: () => setState(createDefault()),
         setMails: ml => setState("mails", ml),
-        setActiveMail: m => setState("activeMail", m)
+        setActiveMail: m => setState("activeMail", m),
+        setInboxActive: () => setState("mailBox", INBOX_NAME),
+        setOutboxActive: () => setState("mailBox", OUTBOX_NAME)
     }
 }
```
Comment (kudanov) [inline src/utils/Store.js]: new line

**Review by kudanov — CHANGES_REQUESTED** [review]
Fix issues

File: src/components/MailList.js (hunk 3 above)
Comment (Akumai) [reply src/components/MailList.js:20]: yep. in a child div

File: src/components/NavPanel/index.js (hunk 4 above)
Comment (Akumai) [reply src/components/NavPanel/index.js:7]: Let me check

Comment (Akumai) [reply src/components/NavPanel/index.js:7]: Nope. Because of sticky btm btn

File: src/components/NavPanel/index.js (hunk 6 above)
Comment (Akumai) [reply src/components/NavPanel/index.js]: done

File: package.json (hunk 5 above)
Comment (Akumai) [reply package.json:40]: yep. It works. done

**Review by kudanov — APPROVED** [review]
Noice

**Pull request merged**